WebRoot/css/zhtml.css

```diff
@@ -492,6 +492,10 @@ A:visited { @Text-link@ }
 	padding: 10px;
 }
 
+.zPrintMsgs .MsgBody-html * {
+	page: auto;
+}
+
```

Comment:
Styles of `@page` defined in an original message will not be applied as side effect, correct?

Reply:
In many case, `@page` style seems to be added by (some?) software/service automatically. The style is applied at print only.
As we discussed, a print page should be printed as we see on a browser. So it is okay to ignore `@page` style by using the `page: auto;` defined here.
I close my question.